client/landing/stepper/declarative-flow/internals/steps-repository/design-setup/index.tsx

```diff
@@ -341,15 +341,20 @@ const designSetup: Step = function DesignSetup( { navigation, flow } ) {
 		window.scrollTo( { top: 0 } );
 	}, [ isForceStaticDesigns ] );
 
-	if ( selectedDesign && isPreviewingDesign && ! showGeneratedDesigns ) {
-		const isBlankCanvas = isBlankCanvasDesign( selectedDesign );
-		const designTitle = isBlankCanvas ? translate( 'Blank Canvas' ) : selectedDesign.title;
-		const shouldUpgrade = selectedDesign.is_premium && ! isPremiumThemeAvailable;
-		const previewUrl = getDesignPreviewUrl( selectedDesign, {
+	let previewUrl = null;
+
+	if ( selectedDesign ) {
+		previewUrl = getDesignPreviewUrl( selectedDesign, {
```

Comment:
@taipeicoder I think this preview URL is not the one that was requested in [this comment](https://github.com/Automattic/wp-calypso/issues/64179#issuecomment-1143733304) because it mentions controls to switch between mobile, tablet and desktop
> Adding a preview link next to the continue CTA so it takes the user into the Preview page where there are controls to switch between mobile, tablet and desktop

What URL would preview the site with those controls?

Reply:
I've left a comment to get more clarification on what the Preview page entails, we'll probably address this in a follow-up PR and scope this one to just scaling the iframe.

Reply:
Yes, I think we have to address the preview in the follow-up PR.

Reply:
Yes you're right!